src/classes/ALLO_AllocationsUtil.cls

```diff
@@ -0,0 +1,1939 @@
+     * @description Actual processing occurs in this method
+     * @param String processName Name of Process Error Records should be associated with
+     * @param Set<Id> oppIds Set of Opportunity Ids to review and process
+     * @param ERR_RecordError recErrs Instance of the ERR_RecordError class.  If populated, will use this.
+     *        If not, it will initialize and retrieve records for this batch
+     * @param Datetime boundary Used to find records modified since last execution
+     * @param Boolean disableAllocationTriggers Indicates whether or not the ALLO_Allocation_TDTM and
+     *        ALLO_PaymentSync_TDTM triggers should be allowed to run
+     * @param Boolean commitAndClearRecordErrors Should we commit and clear record errors here, or delegate
+     *        it to the caller.
+     ********************************************************************************************************/
+    public static Set<Id> processRecords(String processName, Set<Id> oppIds, ERR_RecordError recErrs, Datetime boundary, Boolean disableAllocationTriggers, Boolean commitAndClearRecordErrors) {
+        ALLO_AllocationsUtil.RecordsForReview recs = ALLO_AllocationsUtil.determineRecordsRequiringReview(processName, oppIds, boundary, recErrs);
+
+        if (disableAllocationTriggers) {
+            // Disable Allocation Triggers prior to making any DML Updates
+            ALLO_AllocationsUtil.disableAllocationTriggers();
+        }
+
+        // Will track exclusions across processes
+        ALLO_AllocationsUtil.RecordExclusions recExcl = new ALLO_AllocationsUtil.RecordExclusions();
+
+        // Process Opportunity Allocations
+        ALLO_AllocationsUtil.AllocationResults allocResults = ALLO_AllocationsUtil.processOpportunityAllocations(recs.oppsById, recs.oppsAllocations, recs.campaignAllocations, recs.recurringDonationAllocations, recExcl);
+
+        // Process Opportunity Allocation Results
+        ALLO_AllocationsUtil.processAllocationResults(oppIds, ALLO_AllocationsUtil.PROCESS_DEF.OPP_ALLOC_UPD, allocResults, recs, recExcl);
+
+        // Process Payment Allocations
+        allocResults = ALLO_AllocationsUtil.processPaymentAllocations(recs.oppsById, recs.oppsAllocations, recs.paymentAllocations, recExcl);
+
+        // Process Payment Allocations Results
+        ALLO_AllocationsUtil.processAllocationResults(oppIds, ALLO_AllocationsUtil.PROCESS_DEF.PMT_ALLOC, allocResults, recs, recExcl);
+
+        // Determine successfully processed errors
+        Set<Id> successfullyProcessed = ALLO_AllocationsUtil.determineSucessfullyProcessedRecords(oppIds, recExcl);
+
+        if (commitAndClearRecordErrors) {
+            // Clear Retry Pending and add Retry Successful for Records that were successful that have existing errors flagged for Retry
+            recs.recErrs.clearErrors(successfullyProcessed, Label.errErrorRecordRetrySuccessful);
```

Comment:
```suggestion
            recs.recErrs.clearErrors(successfullyProcessed, System.Label.errErrorRecordRetrySuccessful);
```

Also, it seems odd that a label to display a success message would start with `errError`. If this is unique to Allocations, maybe this label is something like `alloSuccesfulRecordRetry`